src/main/java/org/folio/marc4ld/service/ld2marc/mapper/impl/agent/AgentMapper.java

```diff
@@ -34,12 +35,14 @@
 import org.marc4j.marc.DataField;
 import org.marc4j.marc.MarcFactory;
 import org.marc4j.marc.Subfield;
+import org.marc4j.marc.impl.DataFieldImpl;
 
 @RequiredArgsConstructor
 public abstract class AgentMapper implements Ld2MarcMapper {
 
   private static final Set<PredicateDictionary> SUPPORTED_PREDICATES = Set.of(CREATOR, CONTRIBUTOR);
   private static final String RELATION_PREFIX = "http://bibfra.me/vocab/relation/";
+  private static final DataField EMPTY_DATA_FIELD = new DataFieldImpl(EMPTY, SPACE, SPACE);
```

Comment:
Let us use `marcFactory.newDataField(..)` instead of ` new DataFieldImpl(..)`

Reply:
done